eisenhower-matrix/src/main/webapp/style.css

```diff
@@ -46,41 +44,38 @@ h1 {
   font-size: 19px;
   padding: 10px;
   border-radius: 5px;
-}
-
-#inputHolder {
-  margin-top: 2%;
-  font-size: 20px;
+  position: relative;
+  top: -20px;
```

Comment:
I think it'd be interesting to convert a lot of these specific px amounts to **rem** values. If you try out values between 0.5rem and 2rem, you'll see what I mean. It makes the site more responsive and adaptable to different devices (bc 20px might be big for smaller screens) :)